presto-hive/src/test/java/com/facebook/presto/hive/TestHiveDistributedQueries.java

```diff
@@ -28,7 +31,11 @@
 {
     public TestHiveDistributedQueries()
     {
-        super(() -> createQueryRunner(getTables()));
+        super(() -> createQueryRunner(
+                getTables(),
+                ImmutableMap.of("internal-communication.task-communication-protocol", "THRIFT"),
```

Comment:
Does this mean `HTTP` is not tested in this class any more?

Reply:
My original intension is that HTTP has been covered by other `AbstractTestQueries`. So we may relax. But open to having separate integration test.

Reply:
I think the convention these days is to "fork"  `TestHiveDistributedQueries`, such as `TestHiveDistributedQueriesWithOptimizedRepartitioning`, `TestHivePushdownDistributedQueries`, `TestHiveDistributedQueriesWithExchangeMaterialization`. 

I do agree in this case it doesn't seem that necessary (since HTTP has been covered by other tests)... but following the convention doesn't seem to hurt :)